runtimebp/cpu_test.go

```diff
@@ -59,6 +59,10 @@ func TestReadNumberFromFile(t *testing.T) {
 			Content: "123.456",
 			Error:   true,
 		},
+		"negative": {
+			Content:  "-1",
+			Expected: -1,
+		},
 	}
```

Comment:
> @kylelemons The min should still be 1, so anything less than 1000m should get 1 GOMAXPROCS. That's the best we can do with POSIX thread granularity.

@superq Can you add a test case that covers this case?  It's not clear to me how we will end up setting GOMAXPROCS to 1 if we have a limit of, say, 500 milli cpu.  Does that show up as 5000 per 10 or something?  If so, then we'll return 0, not 1.

Reply:
As @fishy said, this function just returns the cpus/mllicpu as a float. The `runtimebp.defaultMaxProcsFormula()` function does the default rounding up.